sections/ui_fullTable.R

```diff
@@ -36,6 +37,16 @@ body_fullTable <- dashboardBody(
         )
       ),
       width = 12
+    ),
+    sliderInput(
```

Comment:
I would add a fluidRow(column()) here with style = 'padding-left:15px; padding-right:15px;' (similar as in ui_overview.R). Then the whole slider is visible and not cut at the side.
Note that .legend css (line 3) needs to be set to margin-left: -25px; & tags$h5("Wachstumsrate", style = "margin-left: 15px;") (line 28) so that the legend is aligned.

Reply:
ok